internal/featuresupport/support_levels_list.go

```diff
@@ -0,0 +1,86 @@
+package featuresupport
+
+import (
+	"github.com/openshift/assisted-service/internal/usage"
+	"github.com/openshift/assisted-service/models"
+)
+
+func usageNameToID(key string) string {
+	return usage.UsageNameToID(key)
+}
+
+var SupportLevelsList = models.FeatureSupportLevels{
+	&models.FeatureSupportLevel{
+		OpenshiftVersion: "4.6",
+		Features: []*models.FeatureSupportLevelFeaturesItems0{
+			// Tech-Preview features
+			{
+				FeatureID:    usageNameToID(usage.VipDhcpAllocationUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelTechPreview,
+			},
+			// Unsupported features
+			{
+				FeatureID:    usageNameToID(usage.HighAvailabilityModeUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+			{
+				FeatureID:    usageNameToID(usage.CPUArchitectureARM64),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+			{
+				FeatureID:    usageNameToID(usage.UserManagedNetworkWithVMs),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+		},
+	},
+	&models.FeatureSupportLevel{
+		OpenshiftVersion: "4.8",
+		Features: []*models.FeatureSupportLevelFeaturesItems0{
+			// Tech-Preview features
+			{
+				FeatureID:    usageNameToID(usage.VipDhcpAllocationUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelTechPreview,
+			},
+			// Dev-Preview features
+			{
+				FeatureID:    usageNameToID(usage.HighAvailabilityModeUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
+			},
+
+			// Unsupported features
+			{
+				FeatureID:    usageNameToID(usage.CPUArchitectureARM64),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+			{
+				FeatureID:    usageNameToID(usage.UserManagedNetworkWithVMs),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+		},
+	},
+	&models.FeatureSupportLevel{
+		OpenshiftVersion: "4.9",
+		Features: []*models.FeatureSupportLevelFeaturesItems0{
+			// Supported
+			{
+				FeatureID:    usageNameToID(usage.HighAvailabilityModeUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
+			},
+			// Tech-Preview features
+			{
+				FeatureID:    usageNameToID(usage.VipDhcpAllocationUsage),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelTechPreview,
+			},
+			// Dev-Preview
+			{
+				FeatureID:    usageNameToID(usage.CPUArchitectureARM64),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
+			},
+			// Unsupported features
+			{
+				FeatureID:    usageNameToID(usage.UserManagedNetworkWithVMs),
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelUnsupported,
+			},
+		},
+	},
+}
```

Comment:
I think that the better implementation is to define it in an external JSON and read it the same as we do with OpenShift Version and support level. This allow also to use it in python scripts and have different definition to production and staging. Also it is easier to maintain

Reply:
What happened with features that are not on the list?

Reply:
Not listed = supported.